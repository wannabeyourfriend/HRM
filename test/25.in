3
8
inbox
copyto 0
inbox
copyto 1
copyfrom 0
add 1
copyto 0
jump 5