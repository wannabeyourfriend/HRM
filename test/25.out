Fail with execution times 100001